lib/pry/config.rb

```diff
@@ -4,6 +4,158 @@ class Pry
   class Config < Pry::BasicObject
     include Behavior
 
+    # rubocop:disable Metrics/MethodLength
+    def self.defaults
+      defaults = from_hash(
+        input: Pry.lazy_one do
+          lazy_readline
+        end,
+        output: Pry.lazy_one do
+                  $stdout.tap { |out| out.sync = true }
+                end,
+        commands: Pry.lazy_one do
```

Comment:
This is the recommended rubocop style, i thought `Pry.lazy_one { }` read better tbh.